### ISSUE TITLE
Compression level zero

### ISSUE THREAD

**Issue by CarlosRuiz-globalqss** [MinhasKamal/DownGit#35]
Hi @MinhasKamal - first of all - thanks for this wonderful tool!   I find it extremely useful!

I'm using this link to download a zipped folder for iDempiere translation in German:

https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/bxservice/tbayen.translations/tree/master/data/de_DE&fileName=de_DE&rootDirectory=false

The file returned is perfect for what is required to import the translation in the program.

The problem I see is that the ZIP is using compression level 0, this is, not compressing at all, the files weight 5.3MB, and is the same size of the zip file, at compression level 1 the file size is reduced to 799KB, with level 9 is 666KB.

Is it possible to add a flag maybe for the desired compression level, or compress by default in some useful level?

Regards,

Carlos Ruiz

**Comment by MinhasKamal** [comment]
Hello @CarlosRuiz-globalqss - I feel really happy that you found DownGit useful.

The thing is, when a link of directory is provided, DownGit traverses through that directory and downloads each file separately in uncompressed form from GitHub server. After all the files are downloaded, DownGit simply bundles them together. 

Now, as all files are already downloaded, compressing them now will not reduce network usage in any way. Moreover, this compression procedure will simply add extra processing expense; and the user will have to decompress it again. That is why, I just bundled them without imposing any compression.

**Comment by CarlosRuiz-globalqss** [comment]
Thanks @MinhasKamal  for the fast answer, in my case I download the file not to decompress it, but to upload it into another application, same if somebody wants to download it to send it via email for example, then is required to decompress it, and recompress it with a better compression level.

Regards,

Carlos Ruiz

**Comment by MinhasKamal** [comment]
Yes, I understand your case, shall try to add this in future updates.

**Comment by CarlosRuiz-globalqss** [comment]
Thank you @MinhasKamal - very appreciated.
